test/unit/org/apache/cassandra/db/compaction/CompactionStrategyManagerPendingRepairTest.java

```diff
@@ -48,7 +48,7 @@ public class CompactionStrategyManagerPendingRepairTest extends AbstractPendingR
     @Override
     public String createTableCql()
     {
-        return String.format("CREATE TABLE %s.%s (k INT PRIMARY KEY, v INT) ",
+        return String.format("CREATE TABLE %s.%s (k INT PRIMARY KEY, v INT) WITH COMPACTION={'class':'SizeTieredCompactionStrategy'}",
```

Comment:
similarly in this test... I don't see anything stcs-specific in the test. So how do we know if it's not a problem in UCS?

Reply:
Same answer as above really.